app/presenters/chf_index_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ChfIndexPresenter < Blacklight::IndexPresenter
+  class_attribute :description_max_chars
+  self.description_max_chars = 10
+
+  attr_reader :description_field
+
+  def initialize(*args)
+    super
+    # This is an attempt to not hard-code in that our field is called
+    # `description_tesim`, so that doesn't break if it's changed, we
+    # just look for the "description" field. Not really sure if this is
+    # really less fragile, or if there was a better place to get this.
+    @description_field = configuration.index_fields.values.
+      find { |conf| conf.itemprop == "description" }.
```

Comment:
I am not truncating it in fedora, just upon display. I think that's the right thing to do.  This is the field that is used for display for description. If that were to change, then it would have to change. I don't know of any wy around that. 

If you had a _facet_ value that was hundreds of chars wrong, that would indeed be a problem, but not the one we're dealing with now.